safe/definitions/concepts.py

```diff
@@ -549,6 +563,50 @@
         }
     ],
 }
+concepts['under_5'] = {
+    'group': tr('Vulnerability'),
+    'key': 'under_5',
+    'description': tr(''),
+    'citations': [
+        {
+            'text': tr(''),
+            'link': u''
+        }
+    ],
+}
+concepts['over_60'] = {
+    'group': tr('Vulnerability'),
+    'key': 'over_60',
+    'description': tr(''),
+    'citations': [
+        {
+            'text': tr(''),
+            'link': u''
+        }
+    ],
+}
+concepts['child_bearing_age'] = {
+    'group': tr(''),
```

Comment:
It should be a gender, but I am not sure. See my comment here : https://github.com/inasafe/inasafe/issues/4092#issuecomment-292961495

I will set it to Gender for now.